3rdparty/mbedtls/mbedtls/.travis.yml

```diff
@@ -35,7 +42,7 @@ addons:
   coverity_scan:
     project:
       name: "ARMmbed/mbedtls"
-    notification_email: p.j.bakker@polarssl.org
+    notification_email: simon.butcher@arm.com
```

Comment:
I'm curious if there is a better solution than a hard coded individual e-mail address, perhaps an org? 

The rationale being if Simon leaves the company, can we guarantee the correct person is notified or that their e-mail would continue to be monitored? Thus creating further changes needed.

It seems this has happened before.

Reply:
That's not part of the OE SDK project (it's just a clone of the ARM mbedTLS repo) so I wouldn't worry about it. If/when we move to a git submodule approach for dependencies, then this noise will go away, but in the meantime, you can ignore 72f5a736 as being just a straight clone of the 2.16.2 mbedTLS sources.

Reply:
Sounds good, appreciate the clarification!